policyengine_us/variables/gov/hhs/medicaid/eligibility/categories/infant/is_infant_for_medicaid_fc.py

```diff
@@ -8,8 +8,15 @@ class is_infant_for_medicaid_fc(Variable):
     definition_period = YEAR
 
     def formula(person, period, parameters):
-        ma = parameters(period).gov.hhs.medicaid.eligibility.categories.infant
+        p = parameters(period).gov.hhs.medicaid.eligibility.categories.infant
+
         income = person("medicaid_income_level", period)
-        state = person.household("state_code_str", period)
-        income_limit = ma.income_limit[state]
-        return income < income_limit
+        state_code = person.household(
+            "state_code_str", period
+        )  # e.g. "CA", "NY"
+
+        # **directly index** the income_limit map by the state-code vector
+        income_limit = p.income_limit[state_code]
+
+        # allow exactly at the threshold
```

Comment:
please add test to this for the `is_infant_for_medicaid_fc` is seems like we previously regarded the limit as exclusive which has changed 
I also dont think we added unit testing to `is_infant_for_medicaid_nfc`